### PR TITLE
feat!: getInfo support via new RPC getNetworkInfo and getBlockchainInfo

### PR DESCRIPTION
**Pull request by Alex-Werner**
This PR brings modifications in order to provide support on the getInfo method which is used by `insight-api` via the `/status` endpoint. 

## Features : 

- getInfo has additional property : `subVersion`, `bestBlockHash` and `chain`

## Breaking Changes : 

-   getInfo response has changed, `testnet` is not returned (use `chain` or `network`), as well as `proxy` property.

### CODE REVIEW

**Comment by shumkov** [comment]
@Alex-Werner What breaking changes are?

**Comment by Alex-Werner** [comment]
@shumkov Updated description :)

**Comment by shumkov** [comment]
@Alex-Werner are we gonna release it (Insight API, Insight UI) as a new major version?

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "@dashevo/dashcore-node",
   "description": "Full node with extended capabilities using Dashcore and Dash Core (dashd)",
   "author": "BitPay <dev@bitpay.com>",
-  "version": "5.1.4",
+  "version": "5.1.5",
```
Comment (shumkov) [inline package.json]: Let's bump it to `6.0.0`

Comment (Alex-Werner) [reply package.json]: @shumkov Bumped !

**Review by shumkov — APPROVED** [review]
👍

**Pull request merged**